Codeinwp/neve
# Feat - Toolset integration

### Summary
Added toolset integration file ( toolset-config.json )

### Will affect the visual aspect of the product
NO

### Test instructions
- Download the toolset plugin and activate it ( https://handbook.vertistudio.com/3rd-party-products/ )
- Go to Toolset -> Content Templates and create a template that it's applying on posts and pages
- Check to see if the controls from neve are in the sidebar. It should look like this: https://prnt.sc/rw3ipp
- Publish the template, then go and check the single post or page

Closes #1318.
